src/XMLexport.cpp

```diff
@@ -1127,3 +1127,38 @@ bool XMLexport::writeKey(TKey* pT)
 
     return (isOk && (!hasError()));
 }
+
+bool XMLexport::writeScriptElement(const QString & script)
+{
+    QString localScript = script;
+    localScript.replace(QChar('\x01'), QStringLiteral("\xFFFC\x2401")); // SOH
+    localScript.replace(QChar('\x02'), QStringLiteral("\xFFFC\x2402")); // STX
+    localScript.replace(QChar('\x03'), QStringLiteral("\xFFFC\x2403")); // ETX
+    localScript.replace(QChar('\x04'), QStringLiteral("\xFFFC\x2404")); // EOT
+    localScript.replace(QChar('\x05'), QStringLiteral("\xFFFC\x2405")); // ENQ
+    localScript.replace(QChar('\x06'), QStringLiteral("\xFFFC\x2406")); // ACK
+    localScript.replace(QChar('\x07'), QStringLiteral("\xFFFC\x2407")); // BEL
+    localScript.replace(QChar('\x08'), QStringLiteral("\xFFFC\x2408")); // BS
+    localScript.replace(QChar('\x0B'), QStringLiteral("\xFFFC\x240B")); // VT
+    localScript.replace(QChar('\x0C'), QStringLiteral("\xFFFC\x240C")); // FF
+    localScript.replace(QChar('\x0E'), QStringLiteral("\xFFFC\x240E")); // SS
+    localScript.replace(QChar('\x0F'), QStringLiteral("\xFFFC\x240F")); // SI
+    localScript.replace(QChar('\x10'), QStringLiteral("\xFFFC\x2410")); // DLE
+    localScript.replace(QChar('\x11'), QStringLiteral("\xFFFC\x2411")); // DC1
+    localScript.replace(QChar('\x12'), QStringLiteral("\xFFFC\x2412")); // DC2
+    localScript.replace(QChar('\x13'), QStringLiteral("\xFFFC\x2413")); // DC3
+    localScript.replace(QChar('\x14'), QStringLiteral("\xFFFC\x2414")); // DC4
+    localScript.replace(QChar('\x15'), QStringLiteral("\xFFFC\x2415")); // NAK
+    localScript.replace(QChar('\x16'), QStringLiteral("\xFFFC\x2416")); // SYN
+    localScript.replace(QChar('\x17'), QStringLiteral("\xFFFC\x2417")); // ETB
+    localScript.replace(QChar('\x18'), QStringLiteral("\xFFFC\x2418")); // CAN
+    localScript.replace(QChar('\x19'), QStringLiteral("\xFFFC\x2419")); // EM
+    localScript.replace(QChar('\x1A'), QStringLiteral("\xFFFC\x241A")); // SUB
+    localScript.replace(QChar('\x1B'), QStringLiteral("\xFFFC\x241B")); // ESC
+    localScript.replace(QChar('\x1C'), QStringLiteral("\xFFFC\x241C")); // FS
+    localScript.replace(QChar('\x1D'), QStringLiteral("\xFFFC\x241D")); // GS
+    localScript.replace(QChar('\x1E'), QStringLiteral("\xFFFC\x241E")); // RS
+    localScript.replace(QChar('\x1F'), QStringLiteral("\xFFFC\x241F")); // US
+    localScript.replace(QChar('\x7F'), QStringLiteral("\xFFFC\x2471")); // DEL
```

Comment:
Bother, a typo slipped into this (and into the corresponding part in the other file) and I used the '__⑱__' {U+2471 Circled Number Eighteen}  by mistake when I intended to use '__␡__' {U+2421 Symbol For Delete} instead. 😊